app/pages/home/section_components.jsx

```diff
@@ -22,22 +22,41 @@ const CenteredContainer = styled.div`
 `;
 
 const SectionHeader = styled.h2`
-    fontSize: 42px;
-    color: ${props => props.theme.highlight};
+    fontSize: 48px;
+    letterSpacing: 5px;
+    color: ${props => props.theme.secondary};
```

Comment:
If you could make this have to color options based on a passed in prop called dark. It would look something like this

color: ${props => (props.dark ? 'white': 'orange')};